events/wc_undead_events.txt

```diff
@@ -1313,6 +1361,260 @@ character_event = {
 	}
 }
 
+#######################################
+# The Creation of a wicker golem
+#######################################
```

Comment:
It doesn't really make sense to create/copy a new event chain. Better relocalize the abomination creation using [custom commands](https://ck2.paradoxwikis.com/Localisation#Custom_commands).

Reply:
Or using [a second description](https://ck2.paradoxwikis.com/Event_modding#Description) if the text is going to be too different.